appveyor.yml

```diff
@@ -41,10 +45,10 @@ install:
     # Install composer and update per matrix
     - appveyor-retry appveyor DownloadFile https://getcomposer.org/composer.phar
     - cd c:\projects\workspace
-    - IF %dependencies%==lowest appveyor-retry composer update --no-suggest --prefer-dist --prefer-lowest
-    - IF %dependencies%==highest appveyor-retry composer update --no-suggest --prefer-dist
+    - IF %dependencies%==current appveyor-retry composer install
+    - IF %dependencies%==highest appveyor-retry composer update --no-suggest --prefer-dist --ignore-platform-reqs
     - composer show
 
 test_script:
     - cd c:\projects\workspace
-    - vendor\bin\phpunit
```

Comment:
I wonder why the cd line above is required @borNfreee?

Reply:
@theofidry not sure, appveyor config was copied and modified from some another OSS project